<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>301</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:0px;
</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_firstname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_signup">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:grey;
font-size:15px;
border-radius:15px;
}
QPushButton:hover
{
background-color:#4CAF50;
color:white;
}
QPushButton:pressed{
margin-left:1%px;

}
</string>
    </property>
    <property name="text">
     <string>S i g n  U p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_regclose">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:Hover
{
text-align:centre;
border:2px solid black;
background-color:rgb(241,92,34);
}
QPushButton:Pressed
{
margin-left:1px;
}
QPushButton{
background-color:transparent;
border-radius:15px;
font-size:20px;
color:black;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_middlename">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
padding-bottom:7px;
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Middle Name(optional)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lastname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
padding-bottom:7px;
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:11px;
color:black;
</string>
    </property>
    <property name="text">
     <string>Please enter the required information</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
padding-bottom:7px;
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font-size:11px;
</string>
    </property>
    <property name="text">
     <string>Select your gender</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_crpassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>create your password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_crusername">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Create your username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_conpassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:rgba(255,255,255,230);
font-size:13px;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>confirm password</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_male">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-size:13px;
</string>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_female">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-size:13px;
</string>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_other">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:Hover
{
text-align:centre;
border:2px solid black;
background-color:rgb(241,92,34);
}
QPushButton:Pressed
{
margin-left:1px;
}
QPushButton{
background-color:transparent;

font-size:20px;
color:black;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
